hpke/hpke.go

```diff
@@ -282,6 +290,7 @@ var (
 	errInvalidKDF             = errors.New("hpke: invalid KDF identifier")
 	errInvalidKEM             = errors.New("hpke: invalid KEM identifier")
 	errInvalidAEAD            = errors.New("hpke: invalid AEAD identifier")
+	errNotAuthenticatedKEM    = errors.New("hpke: not authenticated KEM")
```

Comment:
This error is a bit cryptic. What does it mean?

Reply:
better now?